packages/core/src/utils/security/handlers/apiKey.ts

```diff
@@ -0,0 +1,38 @@
+import { fromNullable, getOrElse, map } from 'fp-ts/lib/Option';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { get } from 'lodash';
+import { SecurityScheme } from './types';
+import { when } from './utils';
+
+export const apiKeyInCookie = {
+  test: ({ type, in: where }: SecurityScheme) => where === 'cookie' && type === 'apiKey',
+  handle: (someInput: unknown, name: string, resource?: unknown) => {
```

Comment:
why is it `unknown` now? I remember these having concrete types

Reply:
They had not real concrete types. They were using generic arguments, true, but we had no real informations about then — effectively behaving like an `unknown` and using lodash functions to access its properties.

I'd rather make this explicit so that one day I can refactor this instead of pretending we're using the generics correctly but we cannot.